api/templates/includes/vote_form.html

```diff
@@ -1,9 +1,20 @@
-<form id="codesy_vote" 
-   action="{% url 'vote-list' %}" 
-   data-method="POST" >
-   {% csrf_token %}
-   <input name = "claim"  type="hidden" value="{{claim.id}}"></iinput>
-   <label >Do you approve this claim?</label>
-   <input id = "approved" type="checkbox"></input>
-   <input type="submit"></input>
-</form>
\ No newline at end of file
+<div class="callout warning">
+   <h3>You haven't voted yet. Do you approve this claim?.</h3>
+   <form id="codesy_vote"
+      class="ajaxSubmit"
+      action="{% url 'vote-list' %}"
+      data-method="POST" >
+      {% csrf_token %}
+      <input name = "claim"  type="hidden" value="{{claim.id}}"></iinput>
+      <div class="switch large">
+         <input class= "switch-input" id="approved" type="checkbox" >
```

Comment:
I think you just need `name="approved"` here to pass the name parameter in the HTTP post.